user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/GQI/Extensions/API_Reference/IGQIOnDestroy.md

```diff
@@ -9,7 +9,7 @@ uid: GQI_IGQIOnDestroy
 Namespace: `Skyline.DataMiner.Analytics.GenericInterface`  
 Assembly: `SLAnalyticsTypes.dll`
 
-The *IGQIOnDestroy* interface is called when the instance object is destroyed, which happens when the session is closed, e.g. in case of inactivity or when all the necessary data has been retrieved. It can for instance be used to end the connection with a database.
+The *IGQIOnDestroy* interface can be implemented for an Ad Hoc data source or Custom Operator to clean up resources after it has been used. It can, for instance, be used to close a database connection.
```

Comment:
Be careful to be consistent with casing. Here it's Custom Operator capitalized where in other places it's not.
(I'm also not sure what the correct way is)

Reply:
Lowercased it, since technical writing did it lowercase before :)